tests/common/unittest_common.cc

```diff
@@ -1077,6 +1077,38 @@ TEST (commonTensorsConfig, capInvalidParam1_n)
   EXPECT_FALSE (gst_tensors_caps_from_config (NULL));
 }
 
+/**
+ * @brief Test for parsing tensor cap.
+ */
+TEST (commonTensorConfig, parseCapInvalidParam_p)
+{
+  GstTensorsConfig config;
+  GstCaps *caps = gst_caps_new_simple ("other/tensors", "format", G_TYPE_STRING,
+      "flexible", "framerate", GST_TYPE_FRACTION, 30, 1, NULL);
+  gst_tensors_config_init (&config);
+  EXPECT_TRUE (gst_tensors_config_from_cap (&config, caps));
+}
+
+/**
+ * @brief Test for parsing tensor cap with invalid param.
+ */
+TEST (commonTensorConfig, parseCapInvalidParam0_n)
+{
+  GstCaps *caps = gst_caps_new_simple ("other/tensor", "format", G_TYPE_STRING,
+      "flexible", "framerate", GST_TYPE_FRACTION, 30, 1, NULL);
+  EXPECT_FALSE (gst_tensors_config_from_cap (NULL, caps));
+}
+
+/**
+ * @brief Test for parsing tensor cap with invalid param.
+ */
+TEST (commonTensorConfig, parseCapInvalidParam1_n)
+{
+  GstTensorsConfig config;
+  gst_tensors_config_init (&config);
+  EXPECT_FALSE (gst_tensors_config_from_cap (&config, NULL));
+}
+
```

Comment:
Let's add a case where
```

  if (!gst_caps_is_fixed (caps)) {
    nns_logw ("GstCaps is not fixed");
    return FALSE;
  }
```
fails.

E.g., will ```other/tensors,format=static``` fail? (type/dimensions are "not fixed")

Reply:
Added negative test case with `other/tensors,format=static`.
Caps with not fixed type/dimensions failed in `gst_tensors_config_from_cap`.

Reply:
well..plz check below case.
GstCaps *caps
      = gst_caps_new_simple ("other/tensors", "format", G_TYPE_STRING, "static", NULL);
gst_caps_is_fixed (caps) will be true.

Reply:
You're right.
I didn't notice that it was failing for other reasons.
Updated it!
```
[ RUN      ] commonTensorConfig.parseUnfixedCaps_n

** (unittest_common:1516921): WARNING **: 14:46:14.467: GstCaps is not fixed
[       OK ] commonTensorConfig.parseUnfixedCaps_n (0 ms)
```